mobile/salto/src/types/price.ts

```diff
@@ -0,0 +1,7 @@
+export type Price = {
```

Comment:
Can you add some doc with exoample of values for each fields.
It's hard to know what is price.symbol vs price.price.currency.
From where I stand both could be the same thing